packages/components/src/checkbox-control/index.tsx

```diff
@@ -95,6 +95,7 @@ export function CheckboxControl(
 	return (
 		<BaseControl
 			__nextHasNoMarginBottom={ __nextHasNoMarginBottom }
+			__associatedWPComponentName="CheckboxControl"
```

Comment:
We're adding the `__associatedWPComponentName` prop only to components that can disable the `__nextHasNoMarginBottom` right?

Reply:
Correct 👍